src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -188,4 +201,14 @@ typedef struct EnHorse {
         ? true                       \
         : false)
 
+/**
+ * Check if the entrance index of the SaveContext is any of the
+ * "Epona escapes from Lon Lon Ranch" Cutscenes in Hyrule field (there are 4)
+ */
+#define HORSE_CHECK_IS_EPONA_JUMP_CUTSCENE(gSaveContext) \
+    (/* HyruleFieldSouthEponaJump */ gSaveContext.entranceIndex == 0x028A || \
+     /* HyruleFieldWestEponaJump  */ gSaveContext.entranceIndex == 0x028E || \
+     /* HyruleFieldEastEponaJump  */ gSaveContext.entranceIndex == 0x0292 || \
+     /* HyruleFieldGateEponaJump  */ gSaveContext.entranceIndex == 0x0476)
```

Comment:
I'm not sure there's much point to this macro when the values are reused anyway like in `func_8006D684` they're checked against individually

There's a way larger amount of work to do about these entrance indices anyway, eventually